### ISSUE TITLE
Raspberry Pi (Bullseye): occ libresign:install --cfssl results in error

### ISSUE THREAD

**Issue by Albusoline** [LibreSign/libresign#1798]
**Describe the bug**
When trying to install cfssl according to the nextcloud hints (Run occ libresign:install --cfssl), it results in the following error:

```
An unhandled exception has been thrown:
Error: Call to undefined method OC\Files\Node\File::nodeExists() in /var/www/nextcloud/apps/libresign/lib/Service/InstallService.php:427
Stack trace:
#0 /var/www/nextcloud/apps/libresign/lib/Service/InstallService.php(378): OCA\Libresign\Service\InstallService->installCfsslArm()
#1 /var/www/nextcloud/apps/libresign/lib/Command/Install.php(65): OCA\Libresign\Service\InstallService->installCfssl()
#2 /var/www/nextcloud/apps/libresign/vendor/symfony/console/Command/Command.php(298): OCA\Libresign\Command\Install->execute()
#3 /var/www/nextcloud/core/Command/Base.php(177): Symfony\Component\Console\Command\Command->run()
#4 /var/www/nextcloud/apps/libresign/vendor/symfony/console/Application.php(1040): OC\Core\Command\Base->run()
#5 /var/www/nextcloud/apps/libresign/vendor/symfony/console/Application.php(301): Symfony\Component\Console\Application->doRunCommand()
#6 /var/www/nextcloud/apps/libresign/vendor/symfony/console/Application.php(171): Symfony\Component\Console\Application->doRun()
#7 /var/www/nextcloud/lib/private/Console/Application.php(214): Symfony\Component\Console\Application->run()
#8 /var/www/nextcloud/console.php(100): OC\Console\Application->run()
#9 /var/www/nextcloud/occ(11): require_once('/var/www/nextcl...')
#10 {main}
```

**To Reproduce**
see above.

**Expected behavior**
CFSSL in version 1.6.4 installed

**Environment information:**
 - OS: Raspian Bullseye, aarch64
 -LibreSign Version **6.3.1**
 - Go Version **1.20.6**
 - Nextcloud Server Version **25.0.9**

**Additional context**
I've tried to install cfssl from github/cloudflare with go, however I'll get only cfssl Version 1.2 (for whatever reason), which is not enough for Libresign in Nextcloud.

Any ideas?

**Comment by vitormattos** [comment]
Could you check again at the newest release of LibreSign?

I made a lot of improvements.

Closing this issue as solved.

If this issue persists, don't hesitate to open a new issue making reference to this.

> [!NOTE]
> ### If you like this app, don't hesitate to help us
> Ways to help this project:
> - Creating a very nice review of this project at:
>   - **social networks** like LinkedIn, Instagram, etc and putting the @LibreSign and @LibreCodeCoop
>   - **AlternativeTo** https://alternativeto.net/software/libresign/
>   - **Nextcloud apps store**: https://apps.nextcloud.com/apps/libresign
> - Sponsoring the development by **GitHub sponsor** https://github.com/sponsors/LibreSign
> - **helping the translations** on [Transifex](https://app.transifex.com/nextcloud/nextcloud/libresign/)
> - contacting us to have **Enterprise support**: https://libresign.coop

**Comment by Albusoline** [comment]
Dear vitormattos.
Thank you for your comment and your work.
Meanwhile I updated php from 7.4.3 to 8.1.
Nextcloud: 29.0.3
LibreSign: 9.1.2

I installed everything by using:
occ libresign:uninstall --all
occ files:scan-app-data libresign
occ libresign:install --all 

Then I rebooted the server.

However, when using occ libresign:configure:check I get
![grafik](https://github.com/LibreSign/libresign/assets/96739565/fdabe0fd-be70-4239-9e1f-17760e901e07)

On the error log in Nextcloud I found multiple of the entries (almost at the same time):
{"reqId":"XLnVSZf790zQyTeut9r7","level":3,"time":"2024-07-07T14:55:32+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"chmod(): Operation not permitted at /var/www/nextcloud/apps/libresign/lib/Service/Install/InstallService.php#599","userAgent":"--","version":"29.0.3.4","data":{"app":"PHP"},"id":"668ab011e63ec"}

Any ideas?
